andela/titan-dev-ah-frontend
# #163518639 setup react router

What does this PR do?
Setup react router version for the application

Description of Task to be completed?
Setup react router for the application
Create a route to the homepage
Create a route to the login page
Create a route for 404 errors pages
How should this be manually tested?
clone the repo and check out the following branch ch-setup-react-router-version-163518639
run yarn start and navigate to /home , / , or /login in your browser
try to open any other URL you want to check if the 404 page is working
Any background context you want to provide?
None

What are the relevant pivotal tracker stories?
#163518639

Screenshots (if appropriate)
None

Questions:
